wger/manager/templates/schedule/overview.html

```diff
@@ -20,11 +20,7 @@
     {% for schedule in schedules %}
     <tr>
         <td>
-            <a href="{% url 'manager:schedule:view' schedule.id %}">{{ schedule.name }}</a>
-
-            {% if schedule.is_active %}
-            <span class="label label-primary">{% trans "active" %}</span>
-            {% endif  %}
+            <a href="{% url 'manager:schedule:view' schedule.id %}">{{ schedule.name }}</a>&nbsp;{% if schedule.is_active %}<span class="label label-primary">{% trans "active" %}</span>{% endif  %}
```

Comment:
Ah, sorry, I see it now, without the one liners there are 2 whitespaces.

I would leave the ifs the way they where, the point of the nbsp is just to make a bit of space and this is not so much that it breaks the layout or anything. Plus having a readable template is important as well! :)